@ndlib/gatsby-theme-marble/src/components/Shared/SearchTools/SearchResults/HitDisplay/HitResult/index.js

```diff
@@ -57,37 +57,33 @@ const HitResult = ({ hit, referal }) => {
 
 export const highlightTitle = (name, highlight) => {
   if (highlight && highlight['name.folded']) {
-    highlight['name.folded'].forEach(nameHighlight => {
-      const nameVanilla = nameHighlight.replace(/(<([^>]+)>)/ig, '')
-      let count = 0
-      name.forEach(sub => {
-        //        sub.includes(nameVanilla) ? name[count] = name[count].replace(nameVanilla, nameHighlight) : null
-        count++
-      })
-    })
+    const nameHighlight = highlight['name.folded'].toString()
+    const nameVanilla = nameHighlight.replace(/(<([^>]+)>)/ig, '')
+    name = name.includes(nameVanilla) ? name.toString().replace(nameVanilla, nameHighlight) : name
   }
   return name
 }
 
 export const highlightCreator = (creator, highlight) => {
   if (highlight && highlight['creator.folded']) {
-    highlight['creator.folded'].forEach(creatorHighlight => {
+    highlight['creator.folded'].map((creatorHighlight) => {
       const creatorVanilla = creatorHighlight.replace(/(<([^>]+)>)/ig, '')
-      let count = 0
-      creator.forEach(sub => {
-        //        sub.includes(creatorVanilla) ? creator[count] = creator[count].replace(creatorVanilla, creatorHighlight) : null
-        count++
+      creator.map((sub, index) => {
```

Comment:
since you doing an assignment on the loop this should be a forEach rather than a map.